storage/server.go

```diff
@@ -0,0 +1,19 @@
+package storage
+
+import (
+	"golang.org/x/net/context"
```

Comment:
x/net/context? type alias for go 1.9+ so you can use stdlib context

Reply:
Ah wait. I thought I still need to use this package. But the alias works the other way around too? AWESOME :tada:

Reply:
https://github.com/golang/net/blob/master/context/go19.go